create-neon/src/shell.ts

```diff
@@ -15,19 +15,38 @@ export default function shell(cmd: string, args: string[], cwd: string): Promise
     reject = rej;
   });
 
-  child.on('exit', (code) => {
+  function deleteNeonDir(): Promise<void> {
+    return fs.rm(cwd, { force: true, recursive: true });
```

Comment:
Using `cwd` instead of a defined variable seems prone to error. What if we failed to changed directories?